tags/all/families.csv

```diff
@@ -1,4 +1,5 @@
 ABeeZee,/Expressive/Calm,91.6
+ABeeZee,/Purpose/Educational,100

```

Comment:
Not sure about this one.

Reply:
Dave originally mentioned it, and it says "ABeeZee is a children's learning font. " in his description, but I see your point here. @davelab6?

Reply:
This is equal to Borel IMO

Reply:
Per discussion I removed this out to a 'Easy Reading' category https://github.com/google/fonts/commit/7460c7d064c7f2efbf9e4760842d6999804d5dd5